apps/scout-agents/.ebextensions/00_env_vars.config

```diff
@@ -0,0 +1,13 @@
+# Do not put any env var or secrets used by the app here. Put in .ebstalk.app.env/cron.env
+# This file should only have
+# - env variables referenced in docker-compose file
+# - variables needed for the .platform/hooks/predeploy/01_pull_secrets.sh script to run
+
+option_settings:
+    aws:elasticbeanstalk:application:environment:
+        COMPOSE_PROJECT_NAME: "prd"
+        COMPOSE_PROFILES: "prd-scout-agents"
+        SERVICE_ENV: "prd"
+        SERVICE_NAME: "scout-agents"
```

Comment:
remember we dont use hyphens in the service name. it should be scoutgameagents for consistency sake